src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -43,15 +43,15 @@ class AnnualIncomeSection extends React.Component {
             <table className="review usa-table-borderless">
               <tbody>
                 <tr>
-                  <td>Children Gross Income:</td>
+                  <td>Gross Annual Income from Employment:</td>
```

Comment:
Oh oops, must have missed/forgotten that decision. I'm just worried about the case of a screenreader if it will be clear that a field is associated with `Spouse`, for example, if it's not in the label text.